csharp/src/Apache.Arrow.Flight/Apache.Arrow.Flight.csproj

```diff
@@ -0,0 +1,18 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.1</TargetFramework>
+    <LangVersion>8.0</LangVersion>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Google.Protobuf" Version="3.14.0" />
+    <PackageReference Include="Grpc" Version="2.33.1" />
```

Comment:
```suggestion
    <PackageReference Include="Grpc.Tools" Version="2.33.1" PrivateAssets="All" />
```

Reply:
Only need the tooling, and only at build.

Reply:
Actually, `Grpc` doesn't bring in `Grpc.Tools`.

Reply:
I removed the reference to Grpc